src/ngx_http_lua_socket_tcp.h

```diff
@@ -92,6 +92,7 @@ struct ngx_http_lua_socket_tcp_upstream_s {
 
 #if (NGX_HTTP_SSL)
     ngx_str_t                        ssl_name;
+    ngx_ssl_t                        *ssl;
```

Comment:
Style: alignment issue. The `ssl` identifier should be aligned, excluding the `*` prefix.